File: Java/Easy/26/26.java
 {
    public int removeDuplicates(int[] nums) {
        if (nums.length == 0) return 0; // so we can use for loop properly

        Arrays.sort(nums);

        int nonDuplicate = 1;
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] != nums[i -1]) {
                nums[nonDuplicate] = nums[i];
                nonDuplicate++;

            }
        }
    return nonDuplicate;
    }
}

File: Go/Easy/26/26.go
func removeDuplicates(nums []int) int {
    if len(nums) == 0 {
        return 0;
    }
    var nonDuplicate int = 1


    for i := 1;  i < len(nums); i++ {
        if nums[i] != nums[i-1] {
            nums[nonDuplicate] = nums[i]
            nonDuplicate++
        }


    }

    return nonDuplicate

}
